azure-pipelines.yml

```diff
@@ -23,10 +23,10 @@ jobs:
   - template: ./ci/azure-pipelines-build-test.yml
 
 # Runs end-to-end tests
```

Comment:
I have commented out the E2E pipeline for now and will fix them in a separate PR. The bootstrap cluster requires Kubernetes >= 1.11 because CRD subresources are only enabled by default from 1.11 onwards. AFAIK, we can't use

Reply:
Fixed the E2E pipeline.